### PR TITLE
IRINA UPDATE FOR FILE

### PR DESCRIPTION
**Pull request by DekalbIrina**
Create a pool request for TEST Purposes

### CODE REVIEW

File: _slides/SearchForm.aspx.vb [hunk 1]
```diff
@@ -0,0 +1,395 @@
+Imports System.Data
+Imports System.Object
+Imports System.Web.Configuration.WebConfigurationManager
+Imports System.Data.SqlClient
+Imports System.Web.UI.WebControls
+'*************************************************************************************************
+' Application   :   MCAWARRANT
+' Author        :   Irina Gindina
+' Date          :   03/12/2013
```
Comment (c4llmeco4ch) [inline _slides/SearchForm.aspx.vb:9]: March 12th is my favorite day of the year!

**Review by c4llmeco4ch — APPROVED** [review]
This looks great, good job!

File: _slides/SearchForm.aspx.vb [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+
+        Try
+            Select Case Me.RadioButtonList1.SelectedIndex
+
+                Case 0
+
+                    'Search by Defendant First and Last Name         
+                    If Me.txtFirstName.Text = "" And Me.txtLastName.Text = "" Then
+                        Me.lblErrorMessage.Visible = True
+                        Me.lblErrorMessage.Text = "Please enter First name or Last Name and click Submit Button."
```
Comment (mcajuste01) [inline _slides/SearchForm.aspx.vb:39]: This makes sense.

**Review by mcajuste01 — APPROVED** [review]
Testing this comment

File: _slides/SearchForm.aspx.vb (hunk 1 above)
Comment (NNSU99) [reply _slides/SearchForm.aspx.vb:9]: finished reviewing....thx! -nn

File: _slides/SearchForm.aspx.vb [hunk 3]
```diff
@@ -0,0 +1,395 @@
+Imports System.Data
+Imports System.Object
+Imports System.Web.Configuration.WebConfigurationManager
+Imports System.Data.SqlClient
+Imports System.Web.UI.WebControls
+'*************************************************************************************************
+' Application   :   MCAWARRANT
+' Author        :   Irina Gindina
```
Comment (lumapathy) [inline _slides/SearchForm.aspx.vb:8]: Reviewed by UL

File: _slides/SearchForm.aspx.vb (hunk 1 above)
Comment (NNSU99) [reply _slides/SearchForm.aspx.vb:9]: it done.

**Review by NNSU99 — APPROVED** [review]

**Review by lumapathy — APPROVED** [review]
Review done by UL. All good.

**Comment by DekalbIrina** [comment]
Delete IRINA Branch for GitHub and Local Code

**Pull request merged**